tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -852,6 +868,25 @@ const RecipeContent = ({ item, craftable, busy, mode, diet }) => {
                   }
                 />
               )}
+              {!item.non_craftable && item.mass_craftable && (
+                <Button
+                  width="30px"
+                  lineHeight={2.5}
+                  align="center"
+                  tooltip={
+                    'Starts infinite(until you step/run out of things to craft) crafting items/food.'
+                  }
+                  tooltipPosition={'top'}
+                  disabled={!craftable || busy}
+                  icon={'repeat'}
+                  iconSpin={busy ? 1 : 0}
```

Comment:
```suggestion
                  iconSpin={busy}
```
Fairly certain `iconSpin` is the same type as `busy` so this can just passed directly.